components/aws/sagemaker/tests/unit_tests/tests/MonitoringSchedule/test_MonitoringSchedule_spec.py

```diff
@@ -0,0 +1,30 @@
+from MonitoringSchedule.src.MonitoringSchedule_spec import (
+    SageMakerMonitoringScheduleSpec,
+)
+
+import unittest
+
+
+class MonitoringScheduleSpecTestCase(unittest.TestCase):
+    REQUIRED_ARGS = [
+        "--region",
+        "us-west-1",
+        "--monitoring_schedule_name",
+        "test",
+        "--monitoring_schedule_config",
+        "{'test': 'test'}",
+    ]
+    INCORRECT_ARGS = ["--empty"]
+
+    def test_minimum_required_args(self):
+        # Will raise an exception if the inputs are incorrect
+        spec = SageMakerMonitoringScheduleSpec(self.REQUIRED_ARGS)
```

Comment:
nit: would it be worth have a test that has 90% of the required args andhas 1 missing so that this test would fail? I think in its current state it only validates that you have manually passed every required arg. But want to insure the raised inception in case where a user would miss one?